src/Datadog.Trace/Logging/DatadogLogging.cs

```diff
@@ -31,7 +31,8 @@ static DatadogLogging()
             try
             {
                 // We use environment variables and not the tracer settings to avoid a startup race condition between the logger and the tracer.
-                if (Environment.GetEnvironmentVariable(ConfigurationKeys.DebugEnabled) == "1")
+                var ddTraceDebugValue = Environment.GetEnvironmentVariable(ConfigurationKeys.DebugEnabled)?.ToLowerInvariant();
+                if (ddTraceDebugValue == "1" || ddTraceDebugValue == "true")
```

Comment:
We have a [`String.ToBoolean()`](https://github.com/DataDog/dd-trace-dotnet/blob/58908e43b3e1d5da3bcd19499709da6b2b259a83/src/Datadog.Trace/ExtensionMethods/StringExtensions.cs#L29-L46) extension method which checks for several different "truthy" values and is not case sensitive. This may require a `using Datadog.Trace.ExtensionMethods` at the top of this file.

```suggestion
                if (ddTraceDebugValue.ToBoolean() == true)
```